src/hotspot/share/classfile/symbolTable.cpp

```diff
@@ -551,10 +550,9 @@ void SymbolTable::verify() {
 
 // Dumping
 class DumpSymbol : StackObj {
-  Thread* _thr;
   outputStream* _st;
 public:
-  DumpSymbol(Thread* thr, outputStream* st) : _thr(thr), _st(st) {}
+  DumpSymbol(outputStream* st) : _st(st) {}
```

Comment:
I bet there used to be a ResourceMark rm(_thr); here once.
There's similar code in stringTable.cpp.  Doesn't seem beneficial to change.